include/enums/enum_eigen_solver_type.h

```diff
@@ -71,8 +71,8 @@ enum PositionOfSpectrum {LARGEST_MAGNITUDE=0,
                          LARGEST_IMAGINARY,
                          SMALLEST_IMAGINARY,
                          TARGET_IMAGINARY,
-
-                         INVALID_Postion_of_Spectrum};
+                         INVALID_Postion_of_Spectrum,
+                         INVALID_POSITION_OF_SPECTRUM};
```

Comment:
Ok, this worries me.  Are there corresponding changes in the code to accept the new enum option and to spit out a deprecated warning if we see the old option?  Ctrl-F isn't finding them for me, even after I expand all the foo_eigen_bar.C diffs.

Reply:
The old enum was never used anywhere, so I didn't really know what else to do to support it?

Reply:
Oh, was this just one of those habits we get into where whenever we make an enum we give it an "INVALID_WHATEVER" option too?  In that case we're good to go.